.gitignore

```diff
@@ -1,3 +1,6 @@
+.idea/
```

Comment:
It is better to add your IDE config files to your global git ignore

Reply:
fixed